### PR TITLE
CSS page templates - convert to markdown

### PR DESCRIPTION
**Pull request by hamishwillee**
Since we're now working in markdownland for CSS. @rachelandrew can you sanity check please - it is "mostly" what was there previously, modified to the newish format I used on API ref and HTTP headers, then further migrated to markdown.

It should be right, unless the HTML format was wildly wrong.

Don't squash!

### CODE REVIEW

File: files/en-us/mdn/structures/page_types/css_property_page_template/index.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+## Formal syntax
+
+{{CSSSyntax}}
+
+## Examples
+
+Fill in a simple example that nicely shows a typical usage of the property, then perhaps some more complex examples (see our guide on how to add [code examples](/en-US/docs/MDN/Structures/Code_examples) for more information).
+
+```js
```
Comment (teoli2003) [inline files/en-us/mdn/structures/page_types/css_property_page_template/index.md:113]:
```suggestion
```css
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
One small change.

File: files/en-us/mdn/structures/page_types/css_property_page_template/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/mdn/structures/page_types/css_property_page_template/index.md:113]: Thanks!

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Let's land this.

**Pull request merged**